src/__tests__/role-helpers.js

```diff
@@ -149,18 +146,18 @@ test('getRoles returns expected roles for various dom nodes', () => {
 })
 
 test('logRoles calls console.log with output from prettyRoles', () => {
-  const {section} = setup()
-  logRoles(section)
+  const {unnamedSection} = setup()
+  logRoles(unnamedSection)
   expect(console.log).toHaveBeenCalledTimes(1)
   expect(console.log.mock.calls[0][0]).toMatchSnapshot()
 })
 
 test('getImplicitAriaRoles returns expected roles for various dom nodes', () => {
-  const {section, h1, form, radio, input} = setup()
+  const {unnamedSection, h1, unnamedForm, radio, input} = setup()
 
-  expect(getImplicitAriaRoles(section)).toEqual(['region'])
+  expect(getImplicitAriaRoles(unnamedSection)).toEqual([])
   expect(getImplicitAriaRoles(h1)).toEqual(['heading'])
-  expect(getImplicitAriaRoles(form)).toEqual(['form'])
+  expect(getImplicitAriaRoles(unnamedForm)).toEqual([])
```

Comment:
Should we add a named form so we can make sure we capture the named one? Same for the section?